src/creeps/Zombie.h

```diff
@@ -6,28 +6,174 @@
 #include "../basic/Entity.h"
 #include "../collision/CollisionHandler.h"
 #include "../inventory/Inventory.h"
+#include "../buildings/Base.h"
 #include <math.h>
+#include <random>
 #include <vector>
+#include <utility>
 #include <SDL2/SDL.h>
 #include "../view/Window.h"
 #include "../basic/Movable.h"
 
-constexpr static int ZOMBIE_HEIGHT = 125;
-constexpr static int ZOMBIE_WIDTH = 75;
+static constexpr int ZOMBIE_VELOCITY = 50;
+static constexpr int ZOMBIE_INIT_HP  = 100;
+static constexpr int ZOMBIE_FRAMES   = 50;
+static constexpr int ZOMBIE_HEIGHT   = 125;
+static constexpr int ZOMBIE_WIDTH    = 75;
+
+// 8 possible directions
+static constexpr int DIR_CAP = 8;
+static constexpr int DIR_R   = 0;
+static constexpr int DIR_RD  = 1;
+static constexpr int DIR_D   = 2;
+static constexpr int DIR_LD  = 3;
+static constexpr int DIR_L   = 4;
+static constexpr int DIR_LU  = 5;
+static constexpr int DIR_U   = 6;
+static constexpr int DIR_RU  = 7;
+
+// Cardinal directions for setting angles
+static constexpr int NORTH     = 0;
+static constexpr int NORTHEAST = 45;
+static constexpr int EAST      = 90;
+static constexpr int SOUTHEAST = 135;
+static constexpr int SOUTH     = 180;
+static constexpr int SOUTHWEST = 225;
+static constexpr int WEST      = 270;
+static constexpr int NORTHWEST = 315;
```

Comment:
For constants like this with a finite number of values that are all related, use an enum class
Unless you want the possibility of comparing an x position to NORTHWEST
Enum class is better readability and enforces scoping on comparisons, preventing the above issue